docs/en/user/repo/index.md

```diff
@@ -35,6 +35,6 @@ If the namespace strategy is set to custom, the namespace field is also mandator
 ![Create Repository](assets/create-repository.png)
 
 ### Repository Information
-The information screen of repositories shows meta data about the repository. Amongst that are descriptions for the different options on how the repository can be used. 
+The information screen of repositories shows meta data about the repository. Amongst that are descriptions for the different options on how the repository can be used. In the heading you can click the namespace to get the list of all repositories for this namespace.
 
 ![Repository Information](assets/repository-information.png)
```

Comment:
Should we extend the documentation for the creation of repositories to mention that "create" and numbers are not a valid namespace name (if custom strategy is used)?